src/sentry/runner/settings.py

```diff
@@ -101,10 +101,14 @@ def configure(ctx, py, yaml, skip_service_validation=False):
     # which aren't common in default system registries
     import mimetypes
     for type, ext in (
-        ('application/json', 'map'), ('application/font-woff', 'woff'),
```

Comment:
This is garbage from the previous auto formatter, the only thing added is the last line for svg.